pubsub/javadsl/src/test/java/com/lightbend/lagom/javadsl/pubsub/PubSubTest.java

```diff
@@ -48,26 +48,23 @@ public static void setup() {
 
   // yeah, the Akka testkit is in need of some Java 8 love
   private void awaitHasSubscribers(PubSubRef<?> ref, boolean expected) {
-    new JavaTestKit(system) {
+    new TestKit(system) {
       {
-        new AwaitCond(Duration.create(10, TimeUnit.SECONDS)) {
-          @Override
-          protected boolean cond() {
-            try {
-              return expected == ref.hasAnySubscribers().toCompletableFuture().get();
-            } catch (Exception e) {
-              return false;
-            }
+        awaitCond(java.time.Duration.ofSeconds(10), () -> {
```

Comment:
Here as well using a `java.time.Duration` results in a :

`java.time.Duration cannot be converted to scala.concurrent.duration.Duration`

but that's also maybe because `akka-testkit` needs to be bumped 2.5.12 since it's when new method signatures with `java.util.Duration` were introduced ? If so please let me know how to do so in a clean way, thanks !

Reply:
It might be better to do the Akka update in a separate pull request, and then rebase this one onto master (or merge master into it) after that one is merged.

Reply:
Got it, I will open a separate PR for the Akka update to 2.5.12! Thanks!